ci/build_setup.sh

```diff
@@ -13,6 +13,8 @@ echo "ENVOY_SRCDIR=${ENVOY_SRCDIR}"
 function setup_gcc_toolchain() {
   export CC=gcc
   export CXX=g++
+  [ -z "$CIRCLECI" ] || export BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS} --copt=--param=ggc-min-expand=40"  
```

Comment:
Can you add a comment on how you figured this param out and what it does?

Reply:
added